pkg/controller/parser/token_parser.go

```diff
@@ -23,15 +27,9 @@ func NewTokens(secret *corev1.Secret) (*Tokens, error) {
 		return nil, err
 	}
 
-	apiToken, err = ExtractToken(secret, _const.DynatraceApiToken)
-	if err != nil {
-		return nil, err
-	}
-
-	paasToken, err = ExtractToken(secret, _const.DynatracePaasToken)
-	if err != nil {
-		return nil, err
-	}
+	//Errors would have been caught by verifySecret
```

Comment:
`verifySecret` already calls `ExtractToken`. If it would throw an error in lines 31 or 32, it would have already thrown one in 26. Calling `ExtractToken` twice for each token is done so I can catch any error in a single line instead of catching it every call.

Reply:
I see. Fair enough.